src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/Messages.java

```diff
@@ -124,28 +137,44 @@ public void warning(DocTreePath path, String key, Object... args) {
     /**
      * Reports a warning message to the doclet's reporter.
      *
-     * @param e an element identifying the declaration whose position should
-     *  to be included with the message
-     * @param key the name of a resource containing the message to be printed
-     * @param args optional arguments to be replaced in the message.
+     * @param e    an element identifying the declaration whose position should
+     *             be included with the message
```

Comment:
I see what you've changed: one of these words needed to go. Either "should" or "to". You chose "to". Unless you did it for semantical reasons, I note that this file uses "to be" in the vast majority of similar cases.